RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Extract.java

```diff
@@ -41,17 +41,23 @@
 
 abstract class Extract {
 
-    protected Case currentCase = Case.getCurrentCase();
-    protected SleuthkitCase tskCase = currentCase.getSleuthkitCase();
+    protected Case currentCase;
+    protected SleuthkitCase tskCase;
     private final Logger logger = Logger.getLogger(this.getClass().getName());
     private final ArrayList<String> errorMessages = new ArrayList<>();
     String moduleName = "";
     boolean dataFound = false;
 
-    Extract() {
+    Extract() {        
     }
 
     void init() throws IngestModuleException {
+        try {
+            currentCase = Case.getOpenCase();
+            tskCase = currentCase.getSleuthkitCase();
+        } catch (NoCurrentCaseException ex) {
+            throw new IngestModuleException("Exception while getting open case.", ex);
```

Comment:
IngestModuleExceptions messages should be created with Message annotations since they are displayed to users.